### PR TITLE
[zh-cn] update firefox 78 for developers

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

Cleanup for old release notes

### CODE REVIEW

File: files/zh-cn/mozilla/firefox/releases/78/index.md [hunk 1]
```diff
@@ -1,86 +1,88 @@
...
 
-#### 网络监控器
+#### 网络监视器
 
-- In the [Network Monitor](/zh-CN/docs/Tools/Network_Monitor/request_list#Network_request_columns), you can now resize the columns of the request list by dragging the column borders anywhere in the table. ([Firefox bug 1618409](https://bugzil.la/1618409))
-- The [request details panel](/zh-CN/docs/Tools/Network_Monitor/request_details) in the Network Monitor has some UX improvements. ([Firefox bug 1631302](https://bugzil.la/1631302), [Firefox bug 1631295](https://bugzil.la/1631295))
-- If a request was blocked, the [request list](/zh-CN/docs/Tools/Network_Monitor/request_list) now shows the reason, such as an add-on, CSP, CORS, or Enhanced Tracking Protection. ([Firefox bug 1555057](https://bugzil.la/1555057), [Firefox bug 1445637](https://bugzil.la/1445637), [Firefox bug 1556451](https://bugzil.la/1556451))
+- 在[网络监视器](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/request_list/index.html#network-request-columns)中，现在可以通过拖动表格中任意位置的列边框来调整请求列表列的大小。（[Firefox bug 1618409](https://bugzil.la/1618409)）
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/78/index.md:26]:
```suggestion
- 在[网络监视器](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/request_list/index.html#network-request-columns)中，现在可以通过拖动表格中任意位置的列边框来调整请求列表的列宽。（[Firefox bug 1618409](https://bugzil.la/1618409)）
```

File: files/zh-cn/mozilla/firefox/releases/78/index.md [hunk 2]
```diff
@@ -1,86 +1,88 @@
...
+- 网络监控器中的[请求详情面板](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/request_details/index.html)对用户体验进行了一些改进。（[Firefox bug 1631302](https://bugzil.la/1631302)、[Firefox bug 1631295](https://bugzil.la/1631295)）
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/78/index.md:27]:
```suggestion
-对网络监控器中的[请求详情面板](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/request_details/index.html)的用户体验进行了一些改进。（[Firefox bug 1631302](https://bugzil.la/1631302)、[Firefox bug 1631295](https://bugzil.la/1631295)）
```

File: files/zh-cn/mozilla/firefox/releases/78/index.md [hunk 3]
```diff
@@ -1,86 +1,88 @@
...
+- 如果请求被阻止，[请求列表](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/request_list/index.html) 现在会显示原因，如附加组件、CSP、CORS 或增强跟踪保护。（[Firefox bug 1555057](https://bugzil.la/1555057)、[Firefox bug 1445637](https://bugzil.la/1445637)、[Firefox bug 1556451](https://bugzil.la/1556451)）
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/78/index.md:28]:
```suggestion
- 如果请求被阻止，[请求列表](https://firefox-source-docs.mozilla.org/devtools-user/network_monitor/request_list/index.html) 现在会显示原因，如附加组件、CSP、CORS 或增强型跟踪保护。（[Firefox bug 1555057](https://bugzil.la/1555057)、[Firefox bug 1445637](https://bugzil.la/1445637)、[Firefox bug 1556451](https://bugzil.la/1556451)）
```

File: files/zh-cn/mozilla/firefox/releases/78/index.md [hunk 4]
```diff
@@ -1,86 +1,88 @@
...
 
 #### 其他工具
 
-- The [Accessibility](/zh-CN/docs/Tools/Accessibility_inspector) inspector is out of beta. You can use it to check for various accessibility issues on your site. ([Firefox bug 1602075](https://bugzil.la/1602075))
-- Uncaught promise errors now provide all details in the Console, including their name and stack. ([Firefox bug 1636590](https://bugzil.la/1636590))
+- [无障碍性](https://firefox-source-docs.mozilla.org/devtools-user/accessibility_inspector/index.html)检查器已发布测试版。你可以用它来检查网站上的各种辅助功能问题。（[Firefox bug 1602075](https://bugzil.la/1602075)）
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/78/index.md:32]:
```suggestion
- [无障碍环境](https://firefox-source-docs.mozilla.org/devtools-user/accessibility_inspector/index.html)检查器已发布测试版。你可以用它来检查网站上的各种辅助功能问题。（[Firefox bug 1602075](https://bugzil.la/1602075)）
```

File: files/zh-cn/mozilla/firefox/releases/78/index.md [hunk 5]
```diff
@@ -1,86 +1,88 @@
...
 
-- The {{JSxRef("RegExp")}} engine [has been updated](https://hacks.mozilla.org/2020/06/a-new-regexp-engine-in-spidermonkey/) and now supports all new features introduced in ECMAScript 2018:
+- {{JSxRef("RegExp")}} 引擎[已更新](https://hacks.mozilla.org/2020/06/a-new-regexp-engine-in-spidermonkey/)，现在支持 ECMAScript 2018 中引入的所有新特性：
 
-  - [Lookbehind assertions](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Assertions) ([Firefox bug 1225665](https://bugzil.la/1225665))
-  - {{JSxRef("RegExp.prototype.dotAll")}} ([Firefox bug 1361856](https://bugzil.la/1361856))
-  - [Unicode property escapes](/zh-CN/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape) ([Firefox bug 1361876](https://bugzil.la/1361876))
-  - [Named capture groups](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Groups_and_Ranges) ([Firefox bug 1362154](https://bugzil.la/1362154))
+  - [后瞻断言](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Assertions)（[Firefox bug 1225665](https://bugzil.la/1225665)）
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/78/index.md:52]:
```suggestion
  - [后行断言](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Assertions)（[Firefox bug 1225665](https://bugzil.la/1225665)）
```

File: files/zh-cn/mozilla/firefox/releases/78/index.md [hunk 6]
```diff
@@ -1,86 +1,88 @@
...
+  - {{JSxRef("RegExp.prototype.dotAll")}}（[Firefox bug 1361856](https://bugzil.la/1361856)）
+  - [Unicode 属性转义](/zh-CN/docs/Web/JavaScript/Reference/Regular_expressions/Unicode_character_class_escape)（[Firefox bug 1361876](https://bugzil.la/1361876)）
+  - [命名捕获组](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Groups_and_Ranges)（[Firefox bug 1362154](https://bugzil.la/1362154)）
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/78/index.md:55]:
```suggestion
  - [具名捕获组](/zh-CN/docs/Web/JavaScript/Guide/Regular_expressions/Groups_and_Ranges)（[Firefox bug 1362154](https://bugzil.la/1362154)）
```

File: files/zh-cn/mozilla/firefox/releases/78/index.md [hunk 7]
```diff
@@ -1,86 +1,88 @@
...
 #### DOM
 
-- The {{DOMxRef("ParentNode.replaceChildren()")}} method has been implemented ([Firefox bug 1626015](https://bugzil.la/1626015)).
+- 已实现 {{DOMxRef("Element.replaceChildren")}} 方法（[Firefox bug 1626015](https://bugzil.la/1626015)）。
 
-#### 服务工作者
+#### Service worker
 
-- [Extended Support Releases (ESR)](https://www.mozilla.org/en-US/firefox/organizations/): Firefox 78 is the first ESR release that supports [Service workers](/zh-CN/docs/Web/API/Service_Worker_API) (and the [Push API](/zh-CN/docs/Web/API/Push_API)). Earlier ESR releases had no support ([Firefox bug 1547023](https://bugzil.la/1547023)).
+- [Extended Support Releases (ESR)](https://www.mozilla.org/zh-CN/firefox/enterprise/)：Firefox 78 是首个支持 [Service worker](/zh-CN/docs/Web/API/Service_Worker_API)（以及 [Push API](/zh-CN/docs/Web/API/Push_API)）的 ESR 版本。早期的 ESR 版本不提供支持（[Firefox bug 1547023](https://bugzil.la/1547023)）。
```
Comment (yin1999) [inline files/zh-cn/mozilla/firefox/releases/78/index.md:68]:
```suggestion
- [延长支持版（ESR）](https://www.mozilla.org/zh-CN/firefox/enterprise/)：Firefox 78 是首个支持 [Service worker](/zh-CN/docs/Web/API/Service_Worker_API)（以及 [Push API](/zh-CN/docs/Web/API/Push_API)）的 ESR 版本。早期的 ESR 版本不提供支持（[Firefox bug 1547023](https://bugzil.la/1547023)）。
```

**Review by yin1999 — COMMENTED** [review]
Some suggestions:

**Review by yin1999 — APPROVED** [review]
The [release issue has been fixed](https://pontoon.mozilla.org/zh-CN/firefox/devtools/client/debugger.properties/?search=scope&string=159932). Let's merge this :)

Thank you @jasonren0403

**Review by Jaimealdon — APPROVED** [review]

**Pull request merged**